update-netty-substitutions.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+
+# Set directories as variables
+SRC_NETTY_RUNTIME="quarkus/extensions/netty/runtime/src/main/java/io/quarkus/netty"
+SRC_GRPC_RUNTIME="quarkus/extensions/grpc-common/runtime/src/main/java/io/quarkus/grpc/common"
+
+DEST_NETTY_RUNTIME="extension/runtime/src/main/java/io/quarkiverse/temporal/nettyhandling"
+DEST_GRPC_RUNTIME="extension/runtime/src/main/java/io/quarkiverse/temporal/grpchandling"
+
+SRC_NETTY_DEPLOYMENT="quarkus/extensions/netty/deployment/src/main/java/io/quarkus/netty/deployment"
+SRC_GRPC_DEPLOYMENT="quarkus/extensions/grpc-common/deployment/src/main/java/io/quarkus/grpc/common/deployment"
+
+DEST_NETTY_DEPLOYMENT="extension/deployment/src/main/java/io/quarkiverse/temporal/deployment/nettyhandling"
+DEST_GRPC_DEPLOYMENT="extension/deployment/src/main/java/io/quarkiverse/temporal/deployment/grpchandling"
+
+# When bumping temporal we need to check if netty was updated and match with Quarkus version
+QUARKUS_VERSION=3.16.3
```

Comment:
Should this be 3.15.4 to match the current Extension version?

Reply:
Nope, as 3.15.4 uses Netty 4.1.118.Final and there we have changes and substitutions for `ResumptionController` which doesn't exist in the netty shaded version, wish they could match...

Reply:
And now we might have an issue with it... as `3.15.4` uses GRPC java 1.65.1 that uses a really old netty version... Quarkus 3.18... 3.19 uses GRPC java 1.69.1... so when we tested we weren't using Quarkus 3.15... the substitutions won't work... so we might need a release targeting 3.15, and another release targeting newer Quarkus versions... as if we fix for 3.15, 3.19 won't work

Reply:
So for Quarkus 3.15 we would need to target quarkus version `3.12.3` for substitutions, and for newer Quarkus `3.16.3` as those versions matches the shaded netty classes

Reply:
no should we move right to 3.19.3 then?

Reply:
yep lets move forward with 3.19 and not worry about 3.15